map-with-geojson/README.md

```diff
@@ -6,7 +6,7 @@ This example demonstrates how to display GeoJSON on a web map using [React](http
 
 Click the button below to create the necessary AWS resources for this sample app to run. It will open the AWS Management Console and initiate the CloudFormation template deployment process.
 
-[![Launch Stack](https://amazon-location-cloudformation-templates.s3.us-west-2.amazonaws.com/cfn-launch-stack-button.svg)](https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName=react-map-with-geojson-example&templateURL=https://amazon-location-cloudformation-templates.s3.us-west-2.amazonaws.com/samples/web-react-map-with-geojson/template.yaml)
+[![Launch Stack](https://amazon-location-cloudformation-templates.s3.us-west-2.amazonaws.com/cfn-launch-stack-button.svg)](https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName=react-map-with-geojson-example&templateURL=https://amazon-location-sample-resources.s3.us-west-2.amazonaws.com/react-map-with-geojson/cloudformation/template.yaml)
```

Comment:
What about this guy?
(https://amazon-location-cloudformation-templates.s3.us-west-2.amazonaws.com/cfn-launch-stack-button.svg)

Should that be switched to, since we don't know where it's coming from?

Reply:
Yes, done :)